src/lib/bootloader/device_map.rb

```diff
@@ -57,11 +57,7 @@ def contain_disk?(disk)
     end
 
     def disks_order
-      grub_devices = @model.grub_devices
-      grub_devices.select! { |d| d.start_with?("hd") }
-      grub_devices.sort_by! { |dev| dev[2..-1].to_i }
-
-      grub_devices.map { |d| @model.system_device_for(d) }
+      sorted_disks.map { |d| @model.system_device_for(d) }
     end
```

Comment:
NP: Maybe i would name it ordered_disks as we used sorted_disks method, but i'm not the best to recommend method names :)